src/pages/markets/index.js

```diff
@@ -109,98 +108,39 @@ const simple_step_content_commodities = [
     },
 ]
 
-const Item = styled.div`
-    padding: 1.2rem 2.4rem;
-    border-bottom: ${(props) =>
-        props.name === props.active_tab ? '2px solid var(--color-red)' : ''};
-    cursor: pointer;
-    z-index: 10;
-
-    h4 {
-        color: var(--color-red);
-        opacity: ${(props) => (props.name === props.active_tab ? '1' : '0.32')};
-        font-weight: ${(props) => (props.name === props.active_tab ? 'bold' : 'normal')};
-    }
-    @media ${device.tabletL} {
-        padding: 1.5rem 3rem;
-
-        ${Header} {
-            font-size: 3rem;
-            width: max-content;
-        }
-    }
-`
-
-const Separator = styled.div`
-    position: absolute;
-    width: 100%;
-    top: -2px;
-    height: 2px;
-    background-color: var(--color-grey-21);
-`
 const TabsContainer = styled(Flex)`
+    background-color: var(--color-grey-23);
+    padding-top: 2rem;
+
     @media ${device.tabletL} {
         justify-content: flex-start;
         overflow: scroll;
         padding-top: 2rem;
```

Comment:
```suggestion
```

Reply:
It will look odd there has to be space between the tab button and hero section

Reply:
But wasn't it already declared in the main styling on line 113?

Reply:
I see. It seems to be redundant. Good catch 👍🏻